components/expenses/Transaction.js

```diff
@@ -91,7 +91,7 @@ class Transaction extends React.Component {
         <LinkCollective collective={originCollective}>{originCollective.name}</LinkCollective>{' '}
         <FormattedMessage
           id="transaction.usingGiftCardFrom"
-          defaultMessage="using a {giftCard} from {collective}"
+          defaultMessage="using a {giftCard} from {collectiveLink}"
```

Comment:
Let's rename the parameter in `values` rather than `defaultMessage` (otherwise we risk to reset all translations). I'm surprised that CI doesn't detect this issue: we also need to update https://github.com/opencollective/opencollective-frontend/blob/838c53f4f693441508cdc146bbb1f57796442483/components/BudgetItemsList.js#L339